migrations/1635887923233-add-endedAt-to-game.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const { config } = require('dotenv');
+const { MongoClient } = require('mongodb');
+
+module.exports.up = (next) => {
+  config();
+
+  MongoClient.connect(process.env.MONGODB_URI, { useUnifiedTopology: true })
+    .then((client) => client.db())
+    .then((db) => db.collection('games'))
+    .then((collection) =>
+      Promise.all([
+        collection,
+        collection.updateMany({}, { $set: { endedAt: { $add: ["$launchedAt", 1000 * 60 * 45]} } }),
+      ]),
+    )
+    .then(() => next());
+};
+
+module.exports.down = (next) => {
```

Comment:
You can get rid of down() method, we don't use down migrations anywhere in this project.

Reply:
Ok, removed it